networkx/algorithms/centrality/tests/test_group_betweenness_centrality.py

```diff
@@ -0,0 +1,24 @@
+"""
+    Tests for Group Betweenness Centrality
+"""
+
+from __future__ import division
+from nose.tools import *
+import networkx as nx
+
+
+class TestGroupBetweennessCentrality:
+    def test_group_betweenness_centrality_1(self):
+        """Group betweenness centrality in P5 for single node group"""
+        G = nx.path_graph(5)
+        C = [1]
+        b = nx.algorithms.centrality.group.group_betweenness_centrality(G, C, weight=None, normalized=False)
```

Comment:
You should only need: ```nx.group_betweenness_centrality```
Using it that way tests that the namespace/import code is working properly.

Reply:
Done.